bff/src/proxy/proxy.ts

```diff
@@ -181,6 +181,32 @@ export function handlePublicRoutes(router: express.Router) : express.Router {
     res.status(200).send();
   });
 
+  // Get latest audit status for blueprint-env
+  router.get('audit/:org/:id/:token', express.json(), async (req: BFFRequest, res: express.Response, next) => {
```

Comment:
`org` will be inferred from the token so it doesn't need to be in the path.

`id` and `token` are ambiguous and need to be named better.

Reply:
renamed to envSha